dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/ClassDumper.java

```diff
@@ -177,16 +176,10 @@ private static boolean isCompatibleClassFileVersion(JavaClass javaClass) {
     return 45 <= classFileMajorVersion && classFileMajorVersion <= 52;
   }
 
-  private static SymbolReferenceSet scanSymbolReferencesInClass(JavaClass javaClass) {
-    SymbolReferenceSet.Builder symbolTableBuilder = SymbolReferenceSet.builder();
-    ImmutableSet.Builder<ClassSymbolReference> classReferences =
-        symbolTableBuilder.classReferencesBuilder();
-    ImmutableSet.Builder<MethodSymbolReference> methodReferences =
-        symbolTableBuilder.methodReferencesBuilder();
-    ImmutableSet.Builder<FieldSymbolReference> fieldReferences =
-        symbolTableBuilder.fieldReferencesBuilder();
+  private static SymbolReferenceMaps.Builder scanSymbolReferences(Path jar, JavaClass javaClass) {
```

Comment:
`findSymbolReferences` or `listSymbolReferences`

Reply:
Updated to findSymbolReferences